### PR TITLE
Fix getting duplicate task in aws_ecs_task table. Closes #725

### LINKED ISSUE #725: Getting duplicate task in aws_ecs_task table.
**Describe the bug**
https://github.com/turbot/steampipe-plugin-aws/issues/699

```
> select * from aws_ecs_task where cluster_name = 'test-cluster'
+--------------+----------------+-------------+---------------------------------------------------------------------------------------+-------------------+------------------------+---------------------------------------------------------+--------------+-------------------
| cluster_name | desired_status | launch_type | task_arn                                                                              | availability_zone | capacity_provider_name | cluster_arn                                             | connectivity | connectivity_at   
+--------------+----------------+-------------+---------------------------------------------------------------------------------------+-------------------+------------------------+---------------------------------------------------------+--------------+-------------------
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/ecb1ef42e0e1445ba02105679fcb6fab | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | <null>       | <null>            
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/1c623704af14498da22f50616cb99aa1 | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | CONNECTED    | 2021-11-02 16:55:3
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/1c623704af14498da22f50616cb99aa1 | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | CONNECTED    | 2021-11-02 16:55:3
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/44e7da5fe8d04394877feead732504fc | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | CONNECTED    | 2021-11-02 16:55:4
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/44e7da5fe8d04394877feead732504fc | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | CONNECTED    | 2021-11-02 16:55:4
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/a266cddb99764814960f4bada2c2805a | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | <null>       | <null>            
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/a266cddb99764814960f4bada2c2805a | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | <null>       | <null>            
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/1c13200904d749e5b4b892d5c34504c9 | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | CONNECTED    | 2021-11-02 16:55:4
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/ecb1ef42e0e1445ba02105679fcb6fab | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | <null>       | <null>            
| test-cluster | RUNNING        | FARGATE     | arn:aws:ecs:us-east-1:533793682495:task/test-cluster/1c13200904d749e5b4b892d5c34504c9 | us-east-1a        | <null>                 | arn:aws:ecs:us-east-1:533793682495:cluster/test-cluster | CONNECTED    | 2021-11-02 16:55:4
+--------------+----------------+-------------+---------------------------------------------------------------------------------------+-------------------+------------------------+---------------------------------------------------------+--------------+-------------------

```

### PR DESCRIPTION
**Pull request by c0d3r-arnab**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: aws/table_aws_ecs_task.go [hunk 1]
```diff
@@ -18,16 +17,11 @@ func tableAwsEcsTask(_ context.Context) *plugin.Table {
...
+			Hydrate:    listEcsTasks,
 			KeyColumns: []*plugin.KeyColumn{
-				{
-					Name:    "cluster_arn",
-					Require: plugin.Optional,
-				},
 				{
 					Name:    "cluster_name",
-					Require: plugin.Optional,
+					Require: plugin.Required,
```
Comment (mildebrandt) [inline aws/table_aws_ecs_task.go:24]: I'm not sure how I feel about requiring a cluster name when querying for tasks. I may want to find all tasks in my account that have over 6GB of memory. 

But in either case, the following doesn't work:
```
> select task_arn, started_at, stopped_at from aws_ecs_task where cluster_name in ('<redacted cluster 1>', '<redacted cluster 2>')
+----------+------------+------------+
| task_arn | started_at | stopped_at |
+----------+------------+------------+
+----------+------------+------------+
```

But this works when querying against `aws_ecs_cluster`:
```
> select title from aws_ecs_cluster where title in ('<redacted cluster 1>', '<redacted cluster 2>')
+-------------------------------------+
| title                               |
+-------------------------------------+
| <redacted cluster 1>                |
| <redacted cluster 2>                |
+-------------------------------------+
```

Comment (bigdatasourav) [reply aws/table_aws_ecs_task.go]: @mildebrandt for this query below, I hope you have the associated tasks in it?
```
> select task_arn, started_at, stopped_at from aws_ecs_task where cluster_name in ('<redacted cluster 1>', '<redacted cluster 2>')```

```
> select task_arn, cluster_name, created_at from aws_ecs_task where cluster_name in ('test-ec2', 'cluster45')
+------------------------------------------------------------------------------------+--------------+---------------------+
| task_arn                                                                           | cluster_name | created_at          |
+------------------------------------------------------------------------------------+--------------+---------------------+
| arn:aws:ecs:us-east-1:632902152528:task/test-ec2/7b4feb2a7438451fa7616018fecd97fa  | test-ec2     | 2021-11-03 07:23:53 |
| arn:aws:ecs:us-east-1:632902152528:task/cluster45/b8057246631a4201995ac5f3e34c88ea | cluster45    | 2021-11-03 07:24:04 |
+------------------------------------------------------------------------------------+--------------+---------------------+
```

Comment (bigdatasourav) [reply aws/table_aws_ecs_task.go]: Also cluster_name will not be a required column.

> select task_arn, cluster_name, created_at, cpu, memory from aws_ecs_task
```
+------------------------------------------------------------------------------------+--------------+---------------------+-----+--------+
| task_arn                                                                           | cluster_name | created_at          | cpu | memory |
+------------------------------------------------------------------------------------+--------------+---------------------+-----+--------+
| arn:aws:ecs:us-east-1: 6329021522343248:task/cluster45/b8057246631a4201995ac5f3e34c88ea | cluster45    | 2021-11-03 07:24:04 | 256 | 512    |
+------------------------------------------------------------------------------------+--------------+---------------------+-----+--------+
```

Comment (bigdatasourav) [reply aws/table_aws_ecs_task.go]: @mildebrandt We have made some changes in the existing code, could you please take the latest pull and then try running your queries and let us if you are getting the expected result.

Comment (mildebrandt) [reply aws/table_aws_ecs_task.go]: All of the queries on my side look good. This is great! Thank you @c0d3r-arnab and @bigdatasourav!

Comment (bigdatasourav) [reply aws/table_aws_ecs_task.go]: @mildebrandt thanks for your feedback, It will be available in today's release.

**Pull request merged**